src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
```

Comment:
You shouldn't render this component if there is no user at all
```suggestion
  user: User;
```